spec/elastic_apm/central_config_spec.rb

```diff
@@ -192,6 +192,7 @@ module ElasticAPM
             request: { headers: { 'Etag': '___etag___' } }
           )
 
+          expect(subject.send(:headers)).to eq('If-None-Match': '___etag___')
```

Comment:
I bet we could add this check to the webmock stub instead for a cleaner approach?

Reply:
I updated the PR, it is now what you were suggesting?